test/test.ts

```diff
@@ -189,6 +190,15 @@ const types = new Map<string, Test>([
 		],
 		typename: TypeName.Generator
 	}],
+	['asyncGenerator', {
+		is: is.asyncGenerator,
+		fixtures: [
+			(async function * () {
```

Comment:
are you sure `function * ()` is the correct formatting here? with the spaces before **and** after the `*`?

if this is the formatting enforces by the tooling, ignore this comment

Reply:
Yes, it's XO! He requests that the format be like this